protected/humhub/modules/content/components/ContentActiveRecord.php

```diff
@@ -93,6 +94,19 @@ class ContentActiveRecord extends ActiveRecord implements ContentOwner, Movable
      */
     protected $streamChannel = 'default';
 
+    /**
+     * Holds an extra create permission by providing one of the following
+     *
+     *  - BasePermission class string
+     *  - Array of type ['class' => '...', 'callback' => '...']
+     *  - Anonymous function
+     *  - BasePermission instance
+     *
+     * @var string permission instance
+     * @since 1.13.0
+     */
+    protected $createPermission = CreatePublicContent::class;
```

Comment:
Maybe `ManageContent` permission as default?

Reply:
Ok, I have changed it to `ManageContent` permission here https://github.com/humhub/humhub/pull/5827/commits/33cca19cc872d6bdde229d426b2d47a60b82fb74#diff-5fea8ec3ab977d4e8e1db5cf83b2509d7cdccff1e612ca16efb88c2a37e18793R108.
I just thought if we have the separate permission with word "Create" then it is more clear to use it, but I also find the permission `CreatePrivateContent`, so correctly to use `CreatePublicContent || CreatePrivateContent`, but if you think `ManageContent` is ok, then we keep it.